### DISCUSSION TITLE
terraform cloud cli token

### DISCUSSION THREAD

**Discussion by sergiofteixeira** [terraform-linters/tflint#1573] [Q&A] [answered]
how to use terraform cloud cli tokens to get private modules? using the github action superlinter. 

also tried the .terraformrc but didnt work

**Comment by bendrucker** [comment, marked as answer]
Not sure how this is related to TFLint, beyond that it expects you to call `terraform init`. 

HashiCorp has an official action that will setup `terraform` for you, including cloud credentials:

https://developer.hashicorp.com/terraform/tutorials/automation/github-actions

Or you could use environment variables:

https://developer.hashicorp.com/terraform/cli/config/config-file#environment-variable-credentials
